config/locales/client.ar.yml

```diff
@@ -2732,7 +2732,6 @@ ar:
       show_toolbar: "عرض شريط أداة الإنشاء"
       hide_toolbar: "إخفاء شريط أداة الإنشاء"
       modal_ok: "حسنًا"
-      modal_cancel: "إلغاء"
```

Comment:
You don't have to delete strings from every lang, just `en`. Translation bot will take care of it in the next run.